weblate/templates/component.html

```diff
@@ -98,7 +98,6 @@
       {% if user_can_edit_component %}
       <li><a href="{% url 'guide' path=object.get_url_path %}">{% trans "Community localization checklist" %}</a></li>
       <li><a href="{% url 'addons' path=object.get_url_path %}">{% trans "Add-ons" %}</a></li>
-      <li><a href="{% url 'settings' path=object.get_url_path %}">{% trans "Settings" %}</a></li>
```

Comment:
This seems wrong.

Reply:
True. It should be other PR. In component overview you can have Settings menu item just right next to other Setttings menu item (from line 106), both pointing to the same url.

Reply:
Ah, I see, fixed that in 76379e559e051536d5617209f9009f20ef6fbd38.